File: src/flight-status/src/FLIGHTSTATUS.Core/Models/Flight.cs
namespace FLIGHTSTATUS.Core.Models;

public class Flight
{
    public string Airport { get; set; } = string.Empty;
    public string FlightNumber { get; set; } = string.Empty;
    public string Airline { get; set; } = string.Empty;
    public string Origin { get; set; } = string.Empty;
    public string Destination { get; set; } = string.Empty;
    public string DepartureTime { get; set; } = string.Empty;
    public string ArrivalTime { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
    public string Gate { get; set; } = string.Empty;
    public string Terminal { get; set; } = string.Empty;
}

File: src/aviation-mcp/src/models/flight.py
from pydantic import BaseModel, ConfigDict
from pydantic.alias_generators import to_camel


class Flight(BaseModel):
    model_config = ConfigDict(populate_by_name=True, alias_generator=to_camel)

    airport: str
    flight_number: str
    airline: str
    origin: str
    destination: str
    departure_time: str
    arrival_time: str
    status: str
    gate: str
    terminal: str
